diff -Nur postgresql-9.1.9/src/pl/plpython/Makefile postgresql-9.1.9.n/src/pl/plpython/Makefile
--- postgresql-9.1.9/src/pl/plpython/Makefile	2013-04-01 20:23:05.000000000 +0200
+++ postgresql-9.1.9.n/src/pl/plpython/Makefile	2013-05-31 23:35:46.000000000 +0200
@@ -24,7 +24,7 @@
 # Darwin (OS X) has its own ideas about how to do this.
 ifeq ($(PORTNAME), darwin)
 shared_libpython = yes
-override python_libspec = -framework Python
+override python_libspec = -lpython2.7
 override python_additional_libs =
 endif
 
