diff -uNr postgresql-9.1.4/src/Makefile.shlib postgresql-9.1.4.new/src/Makefile.shlib
--- postgresql-9.1.4/src/Makefile.shlib	2012-06-01 01:07:09.000000000 +0200
+++ postgresql-9.1.4.new/src/Makefile.shlib	2012-07-03 12:18:19.731276536 +0200
@@ -99,7 +99,7 @@
 
 ifndef soname
 # additional flags for backend modules
-SHLIB_LINK += $(BE_DLLLIBS)
+SHLIB_LINK += $(BE_DLLLIBS) $(MINITAGE_SSL) 
 endif
 
 # For each platform we support shared libraries on, set shlib to the
